kzz7yh-g04217/cod-ve09v3_kzz7yh-g04217.xml

```diff
@@ -257,10 +260,11 @@
             </div>
             <div xml:id="kzz7yh-g04217-Dd1e457">
                <head xml:id="kzz7yh-g04217-Hd1e459">Quaestio 3</head>
+               <head type="question-title">Utrum anima fuerit assumpta mediante gratia.></head>
```

Comment:
```suggestion
               <head type="question-title">Utrum anima fuerit assumpta mediante gratia</head>
```

Reply:
I think you are putting an extra ">" at the end of your question headers; this isn't really a problem because these are just metadata question labels for our own use, but just for neatness of our code we should leave them out.